ports/rmlui/portfile.cmake

```
@@ -0,0 +1,37 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO mikke89/RmlUi
+    REF 4.0
+    SHA512 b1c8dc48ce3c1ef8e53c7e8d0aa830eec5b968bb3ff7cd778067627de649b45c1a72adfcd168d0ca36018455b6b2a740324cec63304ec4e66dd859ef9d64f674
+    HEAD_REF master
+	PATCHES 
+       fix-uwp.patch
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES 
+        lua             BUILD_LUA_BINDINGS
+    INVERTED_FEATURES
+        freetype        NO_FONT_INTERFACE_DEFAULT
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS
+        ${FEATURE_OPTIONS}
+)
+
+vcpkg_install_cmake()
+vcpkg_fixup_cmake_targets(
+    CONFIG_PATH    lib/RmlUi/cmake
+)
+vcpkg_copy_pdbs()

```

@mikke89 This function has been deprecated, unfortunately.
Luckily I've been spending some time this week _also_ trying to create a port for RmlUi. :joy:

Here's what worked for me:
```suggestion
vcpkg_cmake_install()
vcpkg_cmake_config_fixup(
    CONFIG_PATH lib/${PORT}/cmake
)

vcpkg_copy_pdbs()
```

---

(note for the new commands I needed to add `vcpkg-cmake-config` and `vcpkg-cmake` as dependencies in the `vcpkg.json`)

---

Cool, thanks! I'll make sure to get those changes in.